.github/workflows/schedule-durable.yml

```diff
@@ -9,13 +9,18 @@ on:
         type: choice
         description: Durable infrastructure to run tests against
         required: true
-        default: all
+        default: dev
         options:
           - dev
           - qa
           - tnet
           - prod
           - all
+      test_selector:
+        type: string
+        description: Path regex passed to Jest to select which tests to run
+        required: true
+        default: .
```

Comment:
should this be `*`?

Reply:
I would have thought so too hah. When I try it, Jest says unrecognized pattern. Would you mind checking locally as well just to make sure? Just with pnpm directly, passing the pattern using testPathPattern.